qiskit/pulse/instruction_schedule_map.py

```diff
@@ -155,28 +156,41 @@ def get(self,
             The Schedule defined for the input.
         """
         self.assert_has(instruction, qubits)
-        schedule = self._map[instruction].get(_to_tuple(qubits))
-        if isinstance(schedule, ParameterizedSchedule):
-            schedule = schedule.bind_parameters(*params, **kwparams)
-        return schedule
+        schedule_generator = self._map[instruction].get(_to_tuple(qubits))
 
-    def get_parameters(self, instruction: str, qubits: Union[int, Iterable[int]]) -> Tuple[str]:
-        """Return the list of parameters taken by the given instruction on the given qubits.
+        if callable(schedule_generator):
+            return schedule_generator(*params, **kwparams)
+        # otherwise this is just a Schedule
+        return schedule_generator
+
+    def get_unevaluated(self,
+                        instruction: str,
+                        qubits: Union[int, Iterable[int]]
+                        ) -> Union[Schedule, Callable[..., Schedule]]:
```

Comment:
I'm not familiar with the args for `Callable`, what does `[..., Schedule]` mean?

Reply:
See [this for more info](https://docs.python.org/3/library/typing.html#typing.Callable).